python/pyarrow/_csv.pyx

```diff
@@ -1370,14 +1374,18 @@ cdef class WriteOptions(_Weakrefable):
     __slots__ = ()
 
     def __init__(self, *, include_header=None, batch_size=None,
-                 delimiter=None, quoting_style=None):
+                 delimiter=None, eol=None, null_string=None, quoting_style=None):
         self.options.reset(new CCSVWriteOptions(CCSVWriteOptions.Defaults()))
         if include_header is not None:
             self.include_header = include_header
         if batch_size is not None:
             self.batch_size = batch_size
         if delimiter is not None:
             self.delimiter = delimiter
+        if eol is not None:
+            self.eol = eol
+        if null_string is not None:
+            self.null_string = null_string
```

Comment:
> ???
> E   TypeError: expected bytes, NoneType found


It found this glorious typo 🤦. Only have semi-working syntax highlighting and didn't have enough time to figure out how to run the tests locally :/.